src/test-apps/happy/tests/standalone/wdmNext/test_weave_wdm_next_application_key_01.py

```diff
@@ -42,7 +42,7 @@ def test_weave_wdm_next_application_key_01(self):
         wdm_next_args['total_client_count'] = 2
         wdm_next_args['final_client_status'] = 0
         wdm_next_args['timer_client_period'] = 5000
-        wdm_next_args['test_client_iterations'] = 5
+        count = self.test_client_iterations
```

Comment:
Where does this get set the first time. In some instances, I see this explicitly set to '1'; however, in other cases, like this one, it appears to be set out of scope of this review.

Reply:
Thanks Grant for the feedback!
You are right, there are 2 ways of setting the test count:
1. explicitly set to 1
2. setting it from the wdm_next_test_base class in below code, from setUp() function, every test script will call the setUp() function first. 
class weave_wdm_next_test_base(unittest.TestCase):
    def setUp(self):
          self.test_client_iterations = self.__read_wdm_test_config(
               os.path.dirname(os.path.abspath(__file__))+"/wdm_test_config.json", 
               "test_client_iterations")

Ideally, I would like to use 1 for all tests in Travis CI test to run Sanity test only, but some log check required to have 5 test iterations. I did not change any test logic for now.

I will have meeting with @yunhanw-google tomorrow to discuss if we can change the log check requirement, and use same test count for all tests.